.gitattributes

```diff
@@ -0,0 +1,2 @@
+.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
How does this work?  What is `lfs` and why are the filter and merge attributes needed?  Whats the text flag for?

Reply:
discussed in person